src/index.ts

```diff
@@ -1,26 +1,26 @@
 // Copyright 2022 MFB Technologies, Inc.
 
-import { AsyncRequestStatus } from "./enumerations"
+import { AsyncRequestStatusEnum } from "./enumerations"
 
 type INonErrorState = {
   status:
```

Comment:
You could also use the `Exclude` utility type here, I think `Exclude<AsyncRequestStatus,AsyncRequestState.ERROR>` sould work

Reply:
🤦 of course! thanks.